### ISSUE TITLE
Check if our ACE workaround is still needed

### ISSUE THREAD

**Issue by y0014984** [y0014984/Advanced-Equipment#111]
For some assets the ACE3 configs to activate dragging and carrying do not work. Therefore we have a workaround, that enables these features later by script instead of config. This works.

Perhaps the config does not work because some classes do not support the CBA Extended Event Handlers. In ACE3 Mod I found something in the Sitting Framework. There they are using the MACRO XEH_ENABLED in some classes. Perhaps this enables the support for Extended Event Handlers.

Somewhere else I found this. This could also achive the same goal:

```hpp
class EventHandlers
{
	class CBA_Extended_EventHandlers: CBA_Extended_EventHandlers {};
};
```

We should try this to get rid of our workaround.

**Comment by y0014984** [comment]
This is still needed and will be extended and widely used because we need to enable and disable the ACE3 interactions under certain circumstances. So only setting up these features in config isn't sufficient.

**Comment by y0014984** [comment]
solved via #243
